### PR TITLE
Add Ubuntu support

### PR DESCRIPTION
**Pull request by herpiko**

### CODE REVIEW

**Comment by herpiko** [comment]
@herpiko Mintul jempol kalau ok @tuanpembual @ervien

**Review by tuanpembual — APPROVED** [review]
LGTM

**Review by ervinismu — APPROVED** [review]

File: HACKING.md [hunk 1]
```diff
@@ -3,8 +3,9 @@
 ## Requirements
 
 Ensure that you have:
+- Golang
 - Docker
-- These packages installed: `gpg pbuilder debootstrap devscripts python-apt reprepro`
+- These packages installed: `gpg pbuilder debootstrap devscripts curl reprepro`
```
[the remark this answers is not in the document]
Comment (tuanpembual) [reply HACKING.md:8]: LGTM

[the remark this answers is not in the document]
Comment (herpiko) [reply HACKING.md:8]: @ervinismu It is written like that to let dev copy-paste easily into their terminal.

**Pull request merged**